src/features/card/components/CardTypeDialog.tsx

```diff
@@ -0,0 +1,46 @@
+import { CardContext } from "../../../App";
+
+import { CARD_TYPE_LIST } from "features/card/data/cardTypes";
+import { CardType } from "features/card/types/card.type";
+
+export default function CardTypeDialog() {
+  const cardActionRef = CardContext.useActorRef();
+
+  const handleCardTypeClick = (cardType: CardType) => {
+    cardActionRef.send({
+      type: "SET_CARD_INFO",
+      field: "cardType",
+      value: cardType.type,
+    });
+    cardActionRef.send({ type: "TOGGLE", value: false });
+  };
+
+  return (
+    <div className="modal-dimmed">
+      <div className="modal">
+        <div className="flex-center">
+          {CARD_TYPE_LIST.slice(0, 4).map((cardType) => (
+            <div key={cardType.id} className="modal-item-container">
+              <div
+                className={`modal-item-dot ${cardType.color}`}
+                onClick={() => handleCardTypeClick(cardType)}
+              ></div>
```

Comment:
children이 없는 경우는 self-closing tag를 쓰셔도 좋겠네요! 👍
```suggestion
              <div
                className={`modal-item-dot ${cardType.color}`}
                onClick={() => handleCardTypeClick(cardType)}
              />
```